advanced/keyboard-shortcuts.md

```diff
@@ -63,4 +63,15 @@ This is a list of available keyboard shortcuts within the editor user interface.
 
 ## Add custom shortcuts to TinyMCE
 
-Refer to the [addShortcut]({{site.baseurl}}/api/tinymce/tinymce.editor/#addshortcut) and [tinymce.Shortcuts]({{site.baseurl}}/api/tinymce/tinymce.shortcuts/) sections to add custom keyboard shortcuts to {{site.productname}}.
+> **Important**: Adding a custom shortcut with a keyboard combination that conflicts with an existing shortcut will override the existing shortcut.
```

Comment:
Should we mention clashing with TinyMCE AND browser shortcuts?

Reply:
May have been fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/67539b3f222703c4de3af5e10a6c9848d82ea612